charts/px-backup/values.yaml

```diff
@@ -0,0 +1,60 @@
+namespace: px-backup
+
+customeRegistryEnabled: false
+dockerRegistryURL: docker.io
+dockerImageRepoName: portworx
+imagePullSecrets: docregistry-secret
+imagePullPolicy: Always
+
+persistentStorage:
+  enabled: false
+  storageClassName:
+
+nodeAffinityKey: pxc/enabled
+nodeAffinityValue: false
```

Comment:
These shouldn't be configurable, set up the correct values in the specs

Reply:
Removed from values.yaml and added into all templates.
key: pxbackup/enabled
value: false